lib/jsdom/living/helpers/colors.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+
+exports.computeColor = maybeColor => {
```

Comment:
Can you link this to a spec, and make sure it's following the spec algorithm?

Reply:
I can try. I based this off of the observation that colors returned by `getComputedStyles` are in the rgb format, so I'm not currently aware of what spec dictates this. I'm pretty new to reading/finding specs, so do you have any advice which might help me narrow the search?

Reply:
Oh, nice! Also consider this approach:

Starting at https://drafts.csswg.org/cssom/#dom-window-getcomputedstyle , it looks like most of the work is done in https://drafts.csswg.org/cssom/#resolved-value , so you'd want to implement some sort of `function resolvedValue(...) { ... }`, probably with lots of TODOs for the properties you're not currently working on. This would also ensure you get all color-related properties, not just color, fill, and stroke.